<html lang="en"><meta name="viewport" content="width=device-width, initial-scale=1" /><head><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=G-2VYEP6CXDE"></script><script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag("js", new Date());  gtag("config", "G-2VYEP6CXDE");</script><link rel="stylesheet" href="../init.css"></head><body><section class="css-1mzvpcq eo6uwe90"><div class="css-nxgt9g e4rqgur0"><span class="css-jtjir2 e6o8dkj0"><a data-analytics="sidebar:section" href="/science-and-technology/">Science &amp; technology</a></span><span class="css-17r91f e1kr8lla0"> | <!-- -->Tropical diseases</span></div><h1 class="css-1q8n4mn e164j1a30">A field test for malaria resistance</h1><h2 class="css-1uaitzf e14z8lbb0">It will help to save lives, and may slow resistance’s spread</h2></section><img id="myImg" src="../image/20220122_stp003.jpg" width="auto" height="200"><div class="css-1vjzq4s e1prll3w0"><p class="article__body-text article__body-text--dropcap" data-caps="initial"><span data-caps="initial">A</span><small>N ARMS RACE</small> between pharmacologists and malaria parasites has been going on since the mid-19th century, when widespread use of quinine began. Few better illustrations of natural selection exist than the repeated emergence of resistance to such drugs. Even artemisinin, the most recent addition to the arsenal, has already provoked an evolutionary pushback.</p><p class="article__body-text">At the moment, working out which drugs, if any, a particular case of malaria is resistant to means sending a sample to a laboratory for a <small>PCR</small> test. But malaria is most often a problem in poor countries, where such laboratories are scarce, and so is money to pay for tests and to maintain the machines needed to conduct them. A better way for doctors and paramedics in the field to be able to tell, for a particular patient, which drugs the infection is resistant to would thus be welcome. And that may soon be possible, thanks to work by Ron Dzikowski and Eylon Yavin of the Hebrew University in Jerusalem. As they write in <em>ACS Sensors</em>, they have come up with a trick which they think could be turned into a cheap and deployable detector for drug resistance.</p><div class="adComponent_advert__V79Pp adComponent_incontent__Bxd2J adComponent_hidden___o_ZB advert--inline"><div><div class="adComponent_adcontainer__dO1Zm" id="econ-1"></div></div></div><p class="article__body-text">The term “malaria” covers several similar illnesses caused by single-celled parasites of the genus <em>Plasmodium</em>. The deadliest, <em>Plasmodium falciparum</em>, kills around 600,000 people a year, 80% of them children under five. Dr Dzikowski and Dr Yavin therefore focused their attention on this.</p><p class="article__body-text">They knew from research by others that many of the drug-resistant traits in <em>P. falciparum</em> are marked by small changes in the parasite’s <small>DNA</small>, called single-nucleotide polymorphisms (<small>SNPS</small>). These are often copied into the messenger <small>RNA</small> (m<small>RNA</small>) molecules that transfer instructions from <small>DNA</small> to a cell’s protein-making apparatus.</p><p class="article__body-text">Their own field of research concerns molecules called forced-intercalation peptide nucleic acids (<small>FIT</small>-<small>PNA</small>s). These resemble <small>DNA</small> and <small>RNA</small>, but instead of having a sugar-based backbone from which the chemical bases that constitute the genetic code depend, they have a protein-like one. This means they bind more strongly to m<small>RNA</small> than do normal nucleic acids. Add a fluorescent “reporter” molecule, which releases a photon of light when this binding occurs, and the result is a way of testing for the presence of particular m<small>RNA</small>s.</p><h2>No hiding place</h2><p class="article__body-text">To create their resistance assay Dr Dzikowski and Dr Yavin made <small>FIT</small>-<small>PNA</small>s designed to bind to the seven commonest <small>SNP</small>-marked resistance-inducing mutations, adding reporters that glowed red for artemisinin resistance and green to indicate resistance to chloroquine, currently the most widely used antimalarial. They then raised a range of <em>P. falciparum</em> parasites in their laboratory. Some of these were resistant to artemisinin; some to chloroquine; and some to neither.</p><p class="article__body-text">Once the cultures were established, the two researchers incubated them with their newly created <small>FIT</small>-<small>PNA</small>s for 45 minutes. That done, they took samples and put them under a microscope to look for fluorescence. As they hoped would happen, the artemisinin-resistant cultures glowed red, while the chloroquine-resistant ones glowed green. By contrast, when the cultures containing parasites which lacked resistance were tested, no glow was visible.</p><p class="article__body-text">This approach seems something that could be turned easily into a robust testing kit for blood taken in local clinics. No fancy equipment is needed, just a basic light microscope. A patient can then be treated immediately with the appropriate drug, resulting both in a better outcome for the individual and a negation of the evolutionary advantage of drug resistance, thus slowing its spread. A double-whammy, then, from a clever piece of molecular manipulation. <span data-ornament="ufinish">■</span></p><p class="article__body-text"><em>To enjoy more of our mind-expanding science coverage, <a href="./.html">sign up to Simply Science</a>, our weekly newsletter.</em></p></div></body></html>