<html lang="en"><meta name="viewport" content="width=device-width, initial-scale=1" /><head><!-- Global site tag (gtag.js) - Google Analytics --><script async src="https://www.googletagmanager.com/gtag/js?id=G-2VYEP6CXDE"></script><script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag("js", new Date());  gtag("config", "G-2VYEP6CXDE");</script><link rel="stylesheet" href="../init.css"></head><body><section class="css-ps9sjo e1q9n20l0"><div class="css-nxgt9g e1wpftot0"><span class="css-1pipetz e1mwh2nm0"><a data-analytics="sidebar:section" href="/by-invitation/">By Invitation</a></span><span class="css-1wi1zzr e1kwye4o0"> | <!-- -->Health care</span></div><h1 class="css-1bo5zl0 eoacr0f0">Sir John Bell argues for a global genomic surveillance system to thwart pandemics</h1><h2 class="css-4vhs4z ecgqxun0">Tackling covid-19, monkeypox and other diseases is critical. More surveillance and sequencing would help says the professor of medicine</h2></section><img id="myImg" src="../image/20211023_BID001.jpg" width="auto" height="200"><div class="css-1osicwh e15vdjh41"><p class="article__body-text article__body-text--dropcap" data-caps="initial"><span data-caps="initial">O</span><small>UR ABILITY</small> to identify and track pathogens that cause disease outbreaks is flawed. Take covid-19. At the start of the pandemic it took weeks to discover the disease was caused by a new coronavirus. This slowed the development of vaccines and testing platforms. Emerging evolutionary variants caused dismay, even though it is well understood that every virus evolves in this way. And identification of these variants was haphazard as only a few sites with modern surveillance and genetic-based sequencing techniques could do so. Patchy sequencing efforts led to missed opportunities to identify new variants and to understand the evolving epidemic. Fortunately the speed of development of new vaccines that have proved effective against all variants rescued us but it may not always work this way.</p><p class="article__body-text">Countries able to detect new variants, however, were hardly rewarded for doing so. Border controls and economic difficulties followed; equity of access to the vaccines and antivirals that might emerge from shared information did not. Worse, these countries gained notoriety after variants were commonly named after them—despite the lack of globally representative data to demonstrate where the variants had really emerged from. Variants supposedly came from Kent in Britain, India and South Africa. Actually some of these places were just doing more sequencing than others.</p><div class="advert incontent hidden advert--inline"><div><div class="adcontainer" id="econ-1"></div></div></div><p class="article__body-text">What little genomic data were available were bundled together into large global databases. These helped public-health bodies little in individual countries. Data governance arrangements limited ready access to the sequences and the genomic data were not linked to the kind of metadata needed to manage a pandemic. Information on the kinetics of transmission, the geopositioning of cases, the severity of disease related to individual variants and the vaccine status of infected cases were all missing in this system. You cannot see the importance of a mutation without links to the metadata. In other words, there was a dearth of the information necessary to make the best policy decisions. </p><p class="article__body-text">With a stronger reporting system and better technology, we could easily establish a more effective surveillance tool. We will need it. Expect new outbreaks caused by both known and unknown pathogens; there have been at least seven in the past 20 years. In recent weeks, we have seen a worrying increase in an unidentified, and sometimes fatal hepatitis, infecting significant numbers of children mainly in Britain and Northern Ireland. There are also outbreaks of monkeypox in Britain, America, Australia and other countries, and there has been rare transmissions of H3N8, a highly pathogenic avian flu, to children in China. Developing a better genomic surveillance system is no longer optional. </p><p class="article__body-text">The G7, the G20 and others call urgently for the creation of a global surveillance system for pathogens. To track the development of outbreaks and variants it would need to be able to do three things. First, it should track the evolution of a virus, ideally in real time, with representative community sampling across the globe and standardised data, suitable for analysis, deposited in international databases. To deliver meaningful interpretations of the sequence data, every laboratory will need a set of bioinformatic tools to allow rapid analysis of the sequences, ideally without needing the support of specialised scientists. When sequences are aligned and characterised in standard ways, they can provide comparable data for tracking the disease or uncovering it early in the outbreak. The more representative, robust and comparable the data, the stronger our analytical conclusions and the fewer the number of sequences we need to collect. </p><div class="advert incontent hidden advert--inline"><div><div class="adcontainer" id="econ-2"></div></div></div><p class="article__body-text">Second, the system would need to provide rapid and deep characterisation of new pathogen outbreaks. Invariably this does not come from community testing, but hospitals. In them clinicians treat patients suffering from atypical, worrying infections such as severe pneumonias or haemorrhagic fever. When multiple cases appear, thorough interrogation of samples begins. As with <small>SARS</small>-CoV-2, genomic sequencing can often provide a definitive answer far more quickly than traditional microbiology or virology. The availability of tools and algorithms for pathogen-genome sequencing and analysis opens the possibility of their wide distribution, replacing old tools for routine microbiology. This would revolutionise the field and offer the ability to detect novel pathogens very early and respond accordingly. </p><p class="article__body-text">Third, the system should be able to scan the horizon and screen viral species found across the animal kingdom. Cataloguing these will provide a set of pathogens that might jump species into humans for future reference. <small>HIV</small> and Ebola are examples of diseases that have leapt the species barrier. Experts predict an increase in such zoonotic diseases owing to the growing human population’s disruption of ecosystems. </p><p class="article__body-text">The greatest obstacle to the global surveillance system is the lack of international co-operation around data sharing. For it to work, health agencies and governments need governance frameworks for it and access to technology. Any deal would need to protect data sovereignty for individual countries. Scientists, particularly from poor countries, would need assurances that their efforts and research would be recognised appropriately. Most countries want to share data but are likely to walk away from any system in which data are automatically uploaded without prior consent. A system that facilitates sharing by consent is the only one that will work. </p><p class="article__body-text">Another obstacle is the global distribution of sequencing technology and capacity for genomic molecular microbiology. Fortunately, the gadgets needed are getting more mobile: devices the size of a pencil case are now able to provide sequences in a matter of minutes. The kit is also getting cheaper and international financing efforts can get it to countries who otherwise cannot afford it. Internet bandwidth remains a challenge in many places. But with a little support, we have seen countries make great progress: the Gambia, Equatorial Guinea and Sierra Leone have higher rates of sequencing than France or Italy.</p><p class="article__body-text">The world can and must create an effective global surveillance system. It can now be achieved relatively cheaply, it could be distributed equitably around the world and it could save the global community billions in economic costs and save millions of lives. Embedding genomic-pathogen analysis in routine clinical diagnostics will also revolutionise microbiology. The method will be transferable to all communicable diseases—the endemic, the epidemic and the pandemic. If this is done right, we might just pull off the most exciting innovation since the Petri dish.<br/>_______________</p><p class="article__body-text"><i>Sir John Bell is Regius Professor of Medicine at the University of Oxford. He was the president of Britain’s Academy of Medical Sciences in 2006-11 and chair of the Office for Strategic Coordination of Health Research in 2006-16. He advises the British government on covid-19 and the life sciences, and served on the board of directors of Roche from 2001 to 2020.</i></p></div></body></html>